Angular/src/app/data.service.ts

```diff
@@ -1,22 +1,22 @@
 import { Injectable } from '@angular/core';
 
 export class Employee {
-  ID: number;
-  FirstName: string;
-  LastName: string;
-  Prefix: string;
-  Position: string;
-  BirthDate: string;
-  HireDate: string;
-  Notes: string;
-  Address: string;
-  AddressRequired: boolean;
-  StateID: number;
+  ID!: number;
+  FirstName?: string;
+  LastName?: string;
+  Prefix?: string;
+  Position?: string;
+  BirthDate?: string;
+  HireDate?: string;
+  Notes?: string;
+  Address?: string;
+  AddressRequired?: boolean;
+  StateID?: number;
 }
 
 export class State {
-  ID: number;
-  Name: string;
+  ID!: number;
+  Name?: string;
 }
```

Comment:
seems it should be interface instead of class, and then we could get rid of `!` and `?`

Reply:
Well makes sense, I'll double-check this part.  Initially we used our services code from demos: https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/SimpleArray/Angular/Light/
everywhere)

Reply:
Hm, seems we need to update it too

Reply:
Updated this code and added a [card](https://trello.com/c/EI30zz4Y/3834-wg-demos-ts-update-class-to-interfaces-in-angular-demos) for a squad